core/common/src/LocalDate.kt

```diff
@@ -8,6 +8,7 @@ package kotlinx.datetime
 import kotlinx.datetime.format.*
 import kotlinx.datetime.serializers.*
 import kotlinx.serialization.Serializable
+import kotlin.internal.*
```

Comment:
Minor: why this new import is needed (in several files)?

Reply:
It's for `LowPriorityInOverloadResolution`. The import can be removed by writing the FQN in the annotation.

Reply:
Ah, ok, confused it with kotlinx.datetime.internal.